### ISSUE TITLE
Feature Request: command line utility for Moped

### ISSUE THREAD

**Issue by btbytes** [RobertoMachorro/Moped#10]
Other mac text editors like Cot and TextMate have terminal integration that allows you to open a file from the Terminal by typing:

    cot foo.md
    mate bar.py
    subl another.c #for sublime text

This will greatly increase the utility of Moped for many users because Moped can open a file much quicker than either of the above two text editors, and quick open/edit cycle can make huge difference in the usability of a program (

Here's how [cot](https://github.com/coteditor/cot/blob/e749ffbe82bb1db3d3a0d0c4e9a056ad0e7cb130/cot) does it. It is worth considering if we can repurpose this script (Apache2.0 License) to something similar Moped by changing the `APPLICATION_NAME` variable.

**Comment by RobertoMachorro** [comment]
You are right and that is definitely on the list. Thanks for the cot suggestion, worth a try. The direction I've been thinking about is more like TextMates's **rmate**, which uses sockets and would allow remote editing of files as well.

For now, you can try my [poor man solution for this](https://roberto.machorro.net/projects/2019/08/25/moped-released-appstore.html).

**Comment by NerdyDeedsLLC** [comment]
> For now, you can try my [poor man solution for this](https://roberto.machorro.net/projects/2019/08/25/moped-released-appstore.html) - scroll to the bottom of the post.

```SHELL
alias mopedit="open -a Moped $@"
echo 'alias mopedit="open -a Moped $@"'>>~/.bash_profile # (or .zprofile. Pick your poison)

# Usage: 
# mopedit "filename.ext"
#   - or  - 
# mopedit "file1.ext" "file2.ext"
```
First thing I did, post-install. No sense in bloating such a gloriously-lightweight tool, yanno? 
~_^
